### PR TITLE
Added EUR as currency next to canteens name

### PR DESCRIPTION
**Pull request by lchristus**
(Nils)

### CODE REVIEW

File: src/MensaWidget/MWCanteen.class.st [hunk 1]
```diff
@@ -68,14 +86,18 @@ MWCanteen >> addMealsTo: aCanteenMorph [
...
-		textStyle: TextStyle default;
+	^ AlignmentMorph newRow
+		color: MWColorTheme transparentColor;
+		vResizing: #shrinkWrap;
+		hResizing: #spaceFill;
+		addMorph: (StringMorph contents: self class currency);
+		addMorph: self getSpacerMorph;
+		addMorph: (StringMorph contents: self name);
+		addMorph: self getSpacerMorph;
 		yourself
```
Comment (Kangaroo6942) [inline src/MensaWidget/MWCanteen.class.st:101]: Gievn this code an empty canteen would also show the EUR currency. Can we live with that or do i have to make the empty canteen be a nil object (subclass of the canteen) and overwrite this method then?

**Review by Kangaroo6942 — COMMENTED** [review]
Looks good! Just have to merge develop into it beforehand

File: src/MensaWidget/MWCanteen.class.st (hunk 1 above)
Comment (lchristus) [reply src/MensaWidget/MWCanteen.class.st:101]: Me personally I can live with that

Comment (NilsKasper) [reply src/MensaWidget/MWCanteen.class.st:101]: Me too

**Review by NilsKasper — APPROVED** [review]

**Review by Kangaroo6942 — APPROVED** [review]

**Review by NilsKasper — APPROVED** [review]

**Pull request merged**